### PR TITLE
Added KeyWords as Metadata

### PR DESCRIPTION
**Pull request by IamLRBA**
As per issue #59 
cc 
@Jexsie 
@Ndacyayisenga-droid

### CODE REVIEW

File: content/about/index.de.md [hunk 1]
```diff
@@ -2,13 +2,19 @@
 title: "Über uns"
 description: "Eine Übersicht der verschiedenen Bereiche in denen sich OpenElements einbringt sowie unserer Partner und Kunden."
 layout: "about-us"
+keywords:
```
Comment (Jexsie) [inline content/about/index.de.md:5]: `keywords = ["Open Source", "Java", "OSS", "Open Source Unterstützung", "Java Unterstützung"]`

File: content/about/index.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 title: "About us"
 description: "An overview of the engagement of OpenElements and its partners and customers."
 layout: "about-us"
+keywords:
+  - "Open Source"
+  - "Java"
+  - "OSS"
+  - "Open Source Support"
+  - "Java Support"
```
Comment (Jexsie) [inline content/about/index.md:10]: same here

**Comment by IamLRBA** [comment]
@Jexsie , I have made the changes you requested.

File: content/about/index.md (hunk 2 above)
Comment (IamLRBA) [reply content/about/index.md]: Changes done

File: content/about/index.de.md [hunk 3]
```diff
@@ -2,13 +2,14 @@
 title: "Über uns"
 description: "Eine Übersicht der verschiedenen Bereiche in denen sich OpenElements einbringt sowie unserer Partner und Kunden."
 layout: "about-us"
+keywords: ["Open Source", "Java", "OSS", "Open Source Unterstützung", "Java Unterstützung"]
 
+section_intro: "Die OpenElements GmbH wurde im Jahr 2022 von <a class=\"link-purple\" href=\"/de/about-hendrik/\">Hendrik Ebbers</a> gegründet, um ein Unternehmen zu schaffen, das Open Source und die gemeinschaftliche Zusammenarbeit in einem Java-Ökosystem in seinen Fokus nimmt und stärkt."
 
-section_intro: 'Die OpenElements GmbH wurde im Jahr 2022 von <a class="link-purple" href="/de/about-hendrik/">Hendrik Ebbers</a> gegründet, um ein Unternehmen zu schaffen, das Open Source und die gemeinschaftliche Zusammenarbeit in einem Java-Ökosystem in seinen Fokus nimmt und stärkt.'
+section_engagement_title: "Unser Engagement"
```
Comment (Jexsie) [inline content/about/index.de.md:9]: Why is this changing?

**Review by Jexsie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**